W-13114423-rtfctl-enhancements-kt (#1035)
mulesoft/docs-runtime-fabric: modules/ROOT/pages/install-rtfctl.adoc

rtfctl report --namespace <rtf_namespace>
----

> Run rtfctl report tasks in parallel by using the `workers` flag that enables how many tasks to run in parallel. The default value is 10.

Reviewer: ```suggestion
Run rtfctl report tasks in parallel by using the `workers` flag that controls the number of tasks to run in parallel. The default value is 10.
```

> Run rtfctl report tasks in parallel by using the `workers` flag that enables how many tasks to run in parallel. The  The default value is 10.

Reviewer 2: There's an extra "The" in the second sentence.

----

> rtfctl report --workers 10

Reviewer 3: perhaps use a non default value for number of workers? `rtfctl report --workers 12`

Reviewer 3: or better yet just leave it as a numeric placeholder? `rtfctl report --workers <num_of_workers>`

----

> Generate rtfctl report compression faster by using the `use-fast-compression` flag that enables the compression of the result report archive faster in exchange to have a large report file size. The default value is false.

Reviewer: ```suggestion
Faster compression of the tarball generated by `rtfctl report` with the `--use-fast-compression` flag. The default value is false.
```

----
rtfctl report --use-fast-compression=true
----

> Select which application namespaces you want to include in your report besides from the global rtf installation namespace by using the `<rtf_namespace>`. Using this flag enables you to choose what to include in your reports, and returns a faster report. 

Reviewer: ```suggestion
Select which application namespaces to include in the report besides the installation namespace by using the `--namespaces` flag. The flag controls what to include in the reports and can result in faster collection if only selected namespaces are reported. 
```

> Select which application namespaces you want to include in your report besides from the global rtf installation namespace by using the `<rtf_namespace>`. Using this flag enables you choose what to include in your reports, and returns a faster report. 

Reviewer 2: - Should "besides" be removed in the first sentence?
- Add "to" between "you choose" in the second sentence.

If you are in doubt about whether a message represents a problem or not, discuss it with your system administrator or contact MuleSoft support.

== Download Previous rtfctl CLI Versions

> Use the following API rest call to download different rtfctl tool CLI versions through the rtf management service API:

Reviewer: ```suggestion
Use the following rest API call to download different rtfctl tool CLI versions:
```
